### DISCUSSION TITLE
Exporting metadata and pseudocode (2022.1.0

### DISCUSSION THREAD

**Discussion by IceburgLettuce17** [SamboyCoding/Cpp2IL#374] [Q&A]
In CPP2IL 2022.0.7 I was able to not specify anything other than `--game-path` and have the tool export both metadata and pseudocode. In the latest milestone (2022.1.0 pre-release 18), I haven't found a way to export pseudocode  or metadata, it just exports only DummyDLLs, and there's no indication of it in `--list-output-formats` and `--list-processors`.

**Comment by SamboyCoding** [comment]
Correct. Those features have not yet been reimplemented. Hence why this is a pre-release.
